ntoskrnl/mm/ARM3/mminit.c

```diff
@@ -2332,6 +2298,40 @@ MmArmInitSystem(IN ULONG Phase,
         /* Initialize the platform-specific parts */
         MiInitMachineDependent(LoaderBlock);
 
+#if DBG
+        /* The subection PTE format depends on things being 8-byte aligned */
+        ASSERT((sizeof(CONTROL_AREA) % 8) == 0);
+        ASSERT((sizeof(SUBSECTION) % 8) == 0);
```

Comment:
May be https://github.com/vgalnt/reactos/commit/f3ea0933378dc41acaf89d565faff024f181d576
The location of these structures in the memory must also be 8-byte aligned.

Reply:
Yes, that looks good. Will add it, thanks.